horace_core/utilities/split_counts.m

```diff
@@ -0,0 +1,82 @@
+function [chunks, idxs] = split_counts(counts, max_counts)
```

Comment:
Is this explicitly splitting counts now, even if that's its only current use case, or is it a general `split_chunks`?